ext/phar/tar.c

```diff
@@ -1133,13 +1124,13 @@ void phar_tar_flush(phar_archive_data *phar, zend_string *user_stub, bool is_def
 		} else {
 			phar_entry_info newentry = {0};
 
-			newentry.filename = estrndup(".phar/.metadata.bin", sizeof(".phar/.metadata.bin")-1);
-			newentry.filename_len = sizeof(".phar/.metadata.bin")-1;
+			newentry.filename = ZSTR_INIT_LITERAL(".phar/.metadata.bin", false);
```

Comment:
Is it possible to intern those at startup of the extension? (Obviously for a future PR)

Reply:
We could